docker/run.sh

```diff
@@ -8,27 +8,31 @@ cd "$SCRIPTDIR" || exit 1
 
 source ./env.sh
 
-# Create outputs directory if it does not exist
-[[ -d ./outputs ]] || mkdir ./outputs
-
 echo -e "You are using these values:\n"
+echo -e "Container engine:\t${CONTAINER_ENGINE}"
 echo -e "Volumename:\t${VOLUMENAME}"
 echo -e "Invokeai_tag:\t${CONTAINER_IMAGE}"
 echo -e "local Models:\t${MODELSPATH:-unset}\n"
 
-docker run \
+if [[ "${CONTAINER_ENGINE}" == "podman" ]]; then
+   PODMAN_ARGS="--user=appuser:appuser"
+   unset PLATFORM #causes problems
```

Comment:
if PLATFORM causes problems, then maybe runpod is not buildkit compatible?

Reply:
It looks like [podman is stlil behind](https://pythonspeed.com/articles/podman-buildkit/) when it comes to buildkit.  The good news is that podman 4.1.1 has caching mount support for example.  The bad news is debian 11 includes 3.0.1 and even Ubuntu 22.10 (the latest release) uses podman version 3.4.4.  The author of that article even suggests using actual buildkit with podman, then says wait never mind it doesn't work very well.

(Since you have security as a primary concern, I recommend considering trying podman as the container is run and managed by a local user (and the container's user is ALSO a local user.)  So even if someone breaks out of the container's local user to root user, then they break out of the container entirely, they're STILL constrained within a user process.)

Reply:
I run docker rootless and the user in the container runtime is also not having root permissions 🙈

Please try if your problems are resolved when pulling the built image from https://docs.docker.com/engine/security/rootless/ which would be much better than removing all those features from the Dockerfile 😅

Reply:
Okay I'll try it.  I'm also adding a commit with everything discussed so far that I *can* change and not break the build.